internal/log.go

```diff
@@ -2,17 +2,24 @@ package internal
 
 import (
 	"fmt"
-	"log"
 	"path/filepath"
 	"runtime"
 	"strings"
 )
 
 var (
-	Logger      *log.Logger
-	QueryLogger *log.Logger
+	Logger      IErrorLogger
+	QueryLogger IQueryLogger
 )
 
+type IErrorLogger interface {
+	Output(calldepth int, s string) error
+}
+
+type IQueryLogger interface {
+	Printf(format string, v ...interface{})
+}
```

Comment:
Can you please merge IErrorLogger and IQueryLogger into 1 `internal.Logger` interface? Thanks